src/main/java/com/ericsson/ei/handlers/RmqHandler.java

```diff
@@ -207,7 +215,7 @@ public void publishObjectToWaitlistQueue(String message) {
 
     public void close() {
         try {
-            waitlistContainer.destroy();
+//            waitlistContainer.destroy();
```

Comment:
If commented line is not needed please remove.

Reply:
removed